randnum.c

```diff
@@ -0,0 +1,18 @@
+#include<stdio.h>
```

Comment:
Great job! the only suggestion i have is to include text that will prompt the user.